src/js/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * @name Breadcrumbs
+ *
+ * @description
+ *  A breadcrumbs component that will render the appropriate styles for breadcrumbs.
+ *
+ *
+ * @example
+ *  <Breadcrumbs breadcrumbs={breadcrumbs} home={home} />
+ *
+ *
+ */
+
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import Icon from "../Icon/Icon";
+
+const propTypes = {
+  className: PropTypes.string,
+  breadcrumbs: PropTypes.array.isRequired,
+  home: PropTypes.any,
+  id: PropTypes.string,
+  style: PropTypes.object,
+};
+
+const Breadcrumbs: React.FunctionComponent<
+  PropTypes.InferProps<typeof propTypes>
+> = ({ className, breadcrumbs, home, id, style }) => {
+  const renderHome = () => {
+    if (!home) {
+      return;
+    }
+
+    let content = (
+      <Icon
+        className="Icon Icon-home"
+        mods={null}
+        name="home"
+        otherProps={{}}
+        style={{}}
+      />
+    );
+
+    if (home.url) {
+      content = <a href={home.url || "#"}>{content}</a>;
+    }
+
+    return <li>{content}</li>;
+  };
+
+  const renderSeparator = () => {
+    return (
+      <li>
+        <Icon
+          className="Icon Icon-separator"
+          mods={null}
```

Comment:
You can remove this since its set to null.

Reply:
Done!